communication/src/main/java/org/phenotips/messaging/internal/DefaultActionManager.java

```diff
@@ -211,14 +210,7 @@ private String getEmail(DocumentReference userDocument)
      */
     private void setNotified(String patientId, String matchId)
     {
-        List<PatientMatch> successfulMatches = new LinkedList<>();
-        List<PatientMatch> matchesForPatient = this.matchStorageManager.loadMatchesByReferencePatientId(patientId);
-        for (PatientMatch match : matchesForPatient) {
-            if (match.getMatchedPatientId().equals(matchId)) {
-                successfulMatches.add(match);
-                break;
-            }
-        }
+        List<PatientMatch> successfulMatches = this.matchStorageManager.loadMatchesBetweenPatients(patientId, matchId);
```

Comment:
There is a problem here: which servers do we expect here? I.e. is `patientId` a local ID or remote ID? Is `matchId` a local id or remote id?